components/SearchTrackers.vue

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+    data () {
+      return {
+        /**
+         * example trackers list, where the value will be all relevant info for that tracker
+         * There is no need for these value lists to be equal size, some can have more/less pairs
+         */
+        trackers: {
+          "Alpha 01": { description: "Description of Alpha 01", nickname: "Florian" },
+          "Alpha 02": { description: "Description of Alpha 02", nickname: "Bo" },
+          "Alpha 03": { description: "Description of Alpha 03", nickname: "Ben" },
+          "Alpha 04": { description: "Description of Alpha 04", nickname: "Joshua" },
+          "Phone 01": { description: "Description of Phone 01", nickname: "Meg" },
+          "Phone 02": { description: "Description of Phone 02", nickname: "Jonathan", groups: "NCCN, flooding,  kitten stuck in a tree" }
+        }
+      };
+    },
+    mounted () {
+      /**
+       * event triggered on closing the details window, sent from the DetailsPane component
+       */
+      this.$root?.$on("details-closed", () => {
+        this.clearActive();
+      });
+    },
+    methods: {
+      /**
+       * function template in case we want to implement search
+       */
+        performSearch () {
+          console.log("performing search");
+        },
+        /**
+         * event triggered when clicking on a list item
+         * @param event the event that triggered the function
+         * @param details the list of details passed in, see the data()/trackers part
+         */
+        clickListItem (event : MouseEvent, details : Object) {
```

Comment:
Might be interesting to make a interface for those details object, as it will help TypeScript better type everthing

Reply:
Agreed, same comment as above - your call whether we do it in this PR, or do it once we have the 'real' data

Reply:
Alright and same answer as that same comment above